roles/cloudman-boot/defaults/main.yml

```diff
@@ -6,7 +6,8 @@ cm_helm_values: "{{ lookup('env', 'CM_HELM_VALUES') }}"
 cm_skip_cloudman: "{{ lookup('env', 'CM_SKIP_CLOUDMAN')|default('false', true) }}"
 cm_cluster_name: cloudman-cluster
 cm_namespace_name: cloudman
-cm_chart_version: 0.4.0
+cm_chart_version: "{{ lookup('env', 'CM_CHART_VERSION')|default('false', true) }}"
```

Comment:
Shouldn't the default value be empty? and not false?

Reply:
yeh forgot to change from above I'll push the change